lib/multitransport/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
```

Comment:
I think we can just called this transport.js ?

and the worker, lib/transport-worker.js

Reply:
Well, we have `transport()` already in this PR. are you recommending we merge the two things? It's going to be possible!

Reply:
yes. it's trivium but I think we should keep lib flat

Reply:
This is done.